Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
Yes.  It's explicitly documented that that's required if you ever want to use rspec-mocks outside the scope of a single example:

https://relishapp.com/rspec/rspec-mocks/v/3-3/docs/basics/scope

This bug fix doesn't affect `with_temporary_scope` in any way.